### PR TITLE
ACGAN : Remove lines with no effect

### PR DESCRIPTION
**Pull request by farizrahman4u**

### CODE REVIEW

File: examples/mnist_acgan.py [hunk 1]
```diff
@@ -217,19 +216,14 @@ def build_discriminator():
             noise = np.random.uniform(-1, 1, (2 * batch_size, latent_size))
             sampled_labels = np.random.randint(0, 10, 2 * batch_size)
 
-            # we want to fix the discriminator and let the generator train to
-            # trick it
-            discriminator.trainable = False
-
+            # we want to to train the genrator to trick the discriminator
```
Comment (bayesrule) [inline examples/mnist_acgan.py]: double 'to' here sir

**Comment by fchollet** [comment]
@lukedeo can you  comment on this? What were you trying to do. Also, for the record, changing trainability will only take effect after the model is (re)compiled.

**Comment by lukedeo** [comment]
@fchollet @farizrahman4u yeah, these should definitely not be in there... my bad. [This](https://github.com/farizrahman4u/keras/blob/c55894f8fce4ebea5fe8f386b2edc76d81cd922a/examples/mnist_acgan.py#L154) should be the only place where we set the `Model`-in-`Model` `.trainable` param.

**Comment by bobchennan** [comment]
I run the code and this is the result for first 7 epochs:

Testing for epoch 1:
component              | loss | generation_loss | auxiliary_loss
-----------------------------------------------------------------
generator (train)      | 8.22 | 5.90            | 2.32
generator (test)       | 9.62 | 7.31            | 2.30
discriminator (train)  | 1.42 | 0.02            | 1.40
discriminator (test)   | 1.26 | 0.00            | 1.26
Epoch 2 of 20
599/600 [============================>.] - ETA: 2s
Testing for epoch 2:
component              | loss | generation_loss | auxiliary_loss
-----------------------------------------------------------------
generator (train)      | 9.47 | 7.15            | 2.31
generator (test)       | 9.82 | 7.52            | 2.31
discriminator (train)  | 1.24 | 0.00            | 1.24
discriminator (test)   | 1.20 | 0.00            | 1.20
Epoch 3 of 20
599/600 [============================>.] - ETA: 2s
Testing for epoch 3:
component              | loss | generation_loss | auxiliary_loss
-----------------------------------------------------------------
generator (train)      | 9.85 | 7.55            | 2.31
generator (test)       | 9.55 | 7.25            | 2.30
discriminator (train)  | 1.21 | 0.00            | 1.21
discriminator (test)   | 1.19 | 0.00            | 1.19
Epoch 4 of 20
599/600 [============================>.] - ETA: 2s
Testing for epoch 4:
component              | loss | generation_loss | auxiliary_loss
-----------------------------------------------------------------
generator (train)      | 10.11 | 7.81            | 2.31
generator (test)       | 10.01 | 7.71            | 2.30
discriminator (train)  | 1.20 | 0.00            | 1.20
discriminator (test)   | 1.18 | 0.00            | 1.18
Epoch 5 of 20
599/600 [============================>.] - ETA: 2s
Testing for epoch 5:
component              | loss | generation_loss | auxiliary_loss
-----------------------------------------------------------------
generator (train)      | 10.38 | 8.07            | 2.31
generator (test)       | 10.67 | 8.37            | 2.30
discriminator (train)  | 1.19 | 0.00            | 1.19
discriminator (test)   | 1.18 | 0.00            | 1.18
Epoch 6 of 20
599/600 [============================>.] - ETA: 2s
Testing for epoch 6:
component              | loss | generation_loss | auxiliary_loss
-----------------------------------------------------------------
generator (train)      | 10.51 | 8.21            | 2.31
generator (test)       | 11.00 | 8.70            | 2.30
discriminator (train)  | 1.19 | 0.00            | 1.19
discriminator (test)   | 1.18 | 0.00            | 1.18
Epoch 7 of 20
599/600 [============================>.] - ETA: 2s
...
Testing for epoch 20:
component              | loss | generation_loss | auxiliary_loss
-----------------------------------------------------------------
generator (train)      | 13.24 | 10.93           | 2.30
generator (test)       | 12.49 | 10.19           | 2.30
discriminator (train)  | 1.17 | 0.00            | 1.17
discriminator (test)   | 1.17 | 0.00            | 1.17

And all the generated picture is simply black. Is there anything wrong with the training? I used the exact code.

**Comment by fchollet** [comment]
@lukedeo @bobchennan is this tested? Should we modify the example? Should we removed it altogether until this is cleared up?

**Comment by bobchennan** [comment]
I am not sure I used the right library because the program used PIL which is already dead. 
People can install PIL via apt-get but I am working on a cluster and don't have the permission.
I used Pillow instead with pip as suggested in [here](http://stackoverflow.com/questions/20060096/installing-pil-with-pip).
So I am not sure whether the API is the same and maybe @lukedeo can verify is there any problems for my training results.

**Comment by lukedeo** [comment]
I just ran the example as is on TF, TH, and on CPU and GPU and I get results consistent with my original docstring on the file. @fchollet as a second pair of eyes would you mind running it to make sure you get something consistent as well?

**Comment by fchollet** [comment]
@lukedeo looking good to me. Thanks for  checking.

**Comment by bobchennan** [comment]
I think there is some problem with my training. 
I will try to upgrade Keras from 1.1.1 to 1.1.2 as well as Theano and see the result maybe. 

Currently I am using Keras 1.1.1 and Theano-0.9.0.dev4. The version of CuDNN is 4.0.7 and we are using CUDA 7.5.

**Comment by bobchennan** [comment]
After changing the random seed I got similar result.

**Comment by recluze** [comment]
Second @bobchennan ... with the original seed, I got divergence plus black images all around. Changed seed and current run seems to be generating stuff well.
